src/app.scss

```diff
@@ -0,0 +1,45 @@
+@use './theme.scss';
+
+.header {
+  background: theme.$primary-color;
+  height: 200px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 5px 5px theme.$secondary-color;
+}
+
+.logoimage {
```

Comment:
the naming of `logoimage`, `stripeone` and others are containing two words. 
I would suggest to apply camelCase or add dashes between these words:

`.logo-image` or `.logoImage`.

Reply:
okay i thought we only use Camelcase for variables etc and for classes we write everything small. i will fix it :)